client/app/main/main.html

```diff
@@ -26,38 +22,41 @@
 
 <div class='what-is'>
   <div class='container'>
-    <div class='col-md-6 icons'>
-      <span class='glyphicon glyphicon-phone'></span> +
-      <span class='glyphicon glyphicon-book'></span>
-    </div>
-    <div class='col-md-6'>
-      <h3>What is venue?</h3>
-      venue is a platform that allows instructors to give students credit for
-      attending events. An instructor signs on and creates courses and events,
-      students can then use the venue app to upload event submissions, which
-      can contain images or GPS data that verifies a student attended an event.
+    <div class='row'>
+      <div class='col-md-6 icons'>
+        <span class='glyphicon glyphicon-phone'></span><span class='plus-text'> + </span>
+        <span class='glyphicon glyphicon-book'></span>
+      </div>
+      <div class='col-md-6'>
+        <h3>What is venue?</h3>
+        <p class='description'>Venue is a platform that allows instructors to give students credit for
+        attending events. An instructor signs on and creates courses and events,
+        students can then use the venue app to upload event submissions, which
+        can contain images or GPS data that verifies a student attended an event.</p>
+      </div>
     </div>
   </div>
 </div>
 
 <div id='features' class='features'>
   <div class='container'>
-    <div class='col-md-6'>
-      <h3> Features </h3>
-      <ul>
-        <li>Course Creation and Search</li>
-        <li>Course Sections with Multiple Instructors</li>
-        <li>Instructor Dashboard with Activity Feed</li>
-        <li>User Events and Email Notifications</li>
-        <li>Companion iOS and Android Applications</li>
-        <li>Administrator Control Panels</li>
-        <li>Location, Photo and Content Student Submissions</li>
-      </ul>
-    </div>
-    <div class='col-md-6'>
-      <div class='get-started'>
-        Get started with venue today! <br/>
-        <a href='#'> Get a version for your school </a>
+    <div class='row'>
+      <div class='col-md-6'>
+        <h3> Features </h3>
+        <ul>
+          <li>Create courses with multiple sections</li>
+          <li>Student and Instructor dashboards with activity feeds</li>
+          <li>Geo-tagged events and picture uploads</li>
+          <li>Email notifications</li>
+          <li>Companion iOS and Android apps</li>
+          <li>Administrator Control Panels</li>
+          <li>Bulk data downloads</li>
```

Comment:
...bulk data downloads?

I think I liked the old line, maybe an IMO @leej42 second opinion?